src/main/java/nl/bioinf/minorapplicationdesign/ontpillen/model/MedicineDAO/DrugDao.java

```diff
@@ -5,15 +5,44 @@
 
 public interface DrugDao {
 
+    /**
+     * Function that will add a new Drug object to the InMemoryDrugDao.
+     * It will check if the given drug is a DrugSubstance or a DrugGroup and add
+     * it to the correct map accordingly.
```

Comment:
Het is in ons geval een map, maar als de DAO gebaseerd is op een database is het geen map.